File: freecodecamp/2026-01-11.js
function golfScore(par, strokes) {
    if (strokes === 1) {
        return "Hole in one!";
    }

    const diff = strokes - par;

    switch(diff) {
      case -2: return "Eagle";
      case -1: return "Birdie";
      case 0: return "Par";
      case 1: return "Bogey";
      case 2: return "Double bogey";
      default: return "Invalid score";
    }
}

File: freecodecamp/2026-01-11.py
def golf_score(par, strokes):
    if strokes == 1:
        return "Hole in one!"

    diff = strokes - par

    match diff:
        case -2:
            return "Eagle"
        case -1:
            return "Birdie"
        case 0:
            return "Par"
        case 1:
            return "Bogey"
        case 2:
            return "Double bogey"
        case _:
            return "Invalid score"

